docs/capability-types.md

```diff
@@ -0,0 +1,130 @@
+This is a full list of capability types with their properties, entities and units. See the [general information about capabilities](fixture-format.md#capabilities).
+
+### Possible entities and keywords
+
+| Entity | Allowed Units | `-100%` | `-1%` | `0%` | `1%` | `100%`
+| - | - | - | - | - | - | -
+| Speed | `Hz`, `%` | fast reverse | slow reverse | stop | slow | fast
+| RotationSpeed | `Hz`, `%` | fast CCW | slow CW | stop | slow CW | fast CWW
+| Time | `s`, `ms`, `%` | – | – | instant | short | long
+| Distance | `m`, `%` | – | – | – | near | far
+| Brightness | `lm`, `%` | – | – | off | dark | bright
+| ColorTemperature | `K`, `%` | – | – | – | warm | cold
```

Comment:
Have you thought about CTO and CTB? See <https://en.wikipedia.org/wiki/Color_correction#Gel_nomenclature>

Reply:
Good idea. Solution: Change color temperature keywords:
* -100%: warm / CTO
* 0%: default
* 100%: cold / CTB

Reply:
Done.